frontend/src/static/iconfont/iconfont.js

```
@@ -1 +1,70 @@
+          document.write(
+            "<style>.svgfont {display: inline-block;width: 1em;height: 1em;fill: currentColor;vertical-align: -0.1em;font-size:16px;}</style>",
+          );
+        } catch (t) {
+          console?.log(t);
+        }
+      }
+      (i = () => {
+        let t;
+        let e = document.createElement("div");
+        (e.innerHTML = o._iconfont_svg_string_4785399),
+          (e = e.getElementsByTagName("svg")[0]) &&
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)